import etlFunction from './etlFunction'

try {
  etlFunction()
} catch (e) {
  console.log('following error occured')
  console.log(e)
}
